crates/starknet/contracts/artifacts/MockStarknetMessaging.json

```diff
@@ -0,0 +1,2599 @@
+{
```

Comment:
Do we need this json here? we already have this artifact in contracts folder

Reply:
> All solidity contracts are taken from https://github.com/starknet-io/starkgate-contracts/tree/main/src/starkware/starknet/solidity?

Nope, for now I took exactly what you had in the devnet-py. As I didn't know if I was allowed to modify them.

> Can we also document in readme.md how json artifact/artifacts are generated for future developers? Sooner or later this can change and we will need to make an update.

Yes this is planned, everything will be documented once the definitive location/organization of files is approved.

> Do we need this json here? we already have this artifact in contracts folder

This is one comment I made in the code, due to how cargo works, this location of the contracts is relative to the `Cargo.toml` file. So when you execute at workspace level / package level, the resolution is different. I left a comment as this needs to be solved. I am perhaps not using what I need to use, please feel free to point out what I'm surely missing out then. :+1:

Reply:
Path resolution solved in d8bfca76e6402e872f2552a18ef246bae1adf2e2

Reply:
Documentation solved in 0eea51d444ad586125647bfa27a722cbec5a43fc